packages/grafana-ui/src/components/Forms/Form.mdx

```diff
@@ -106,11 +108,38 @@ import { Forms } from '@grafana/ui';
   </>
 )}
 </Forms.Form>
+```
+Note that when using `Forms.InputControl`, it expects the name of the prop that handles input change to be called `onChange`.
+If the property is named differently for any specific component, additional `onChangeName` prop has to be provided, specifying the name.
+Additionally, the value to for the `onChange` prop is passed as an array. Check [react-hook-form docs](https://react-hook-form.com/api/#Controller)
+for more prop options.
```

Comment:
Hmm, this must have change recently, hasn't it?

Reply:
Not to my knowledge, according to the release note's there weren't any changes related to it since the version we're using. 

To clarify - this `onChange` signature is specific to react-hook-form's Controller component, not react-select's.

Reply:
yes, I get that. Interestingly enough, when I have visited the link above few minutes ago it asked me to update and there was definitely a change in the docs last week: https://github.com/react-hook-form/react-hook-form-website/blame/df42aaa867045d35fc23d98b3d798d2ce63a28a6/src/data/en/api.tsx#L966

Reply:
But anyways, good catch with the API! <3